### PR TITLE
Issue#EPR-15:Added form template for x-owner role

### PR DESCRIPTION
**Pull request by aishwa8141**
1)Added check to show the profile page for x-owner in the header component 
2)added cancel button in the update profile page

### CODE REVIEW

File: ui-sample/util-service/templates/X-OwnerFormTemplate.json [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+                },
+                "index": 7
+            },
+            {
+                "code": "role",
+                "dataType": "text",
+                "name": "role",
+                "label": "EkStep role",
+                "description": "EkStep role",
+                "editable": true,
```
Comment (indrajra) [inline ui-sample/util-service/templates/X-OwnerFormTemplate.json]: For an x-owner, none of the fields are editable. Even if we don't expose the buttons, lets also fix this.

File: ui-sample/util-service/templates/X-OwnerFormTemplate.json [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+                "placeholder": "Project name",
+                "inputType": "select",
+                "required": false,
+                "displayProperty": "Editable",
+                "visible": true,
+                "renderingHints": {
+                    "semanticColumnWidth": "four"
+                },
+                "range": [
```
Comment (indrajra) [inline ui-sample/util-service/templates/X-OwnerFormTemplate.json]: Consider making all of this range blank, since it is not going to be edited.

Comment (aishwa8141) [reply ui-sample/util-service/templates/X-OwnerFormTemplate.json]: done

File: ui-sample/util-service/templates/X-OwnerFormTemplate.json (hunk 1 above)
Comment (aishwa8141) [reply ui-sample/util-service/templates/X-OwnerFormTemplate.json]: done

**Pull request merged**